src/Algebra/Graph/Acyclic/AdjacencyMap.hs

```diff
@@ -0,0 +1,449 @@
+module Algebra.Graph.Acyclic.AdjacencyMap (
+  -- * Data structure
+  AdjacencyMap, aam,
```

Comment:
If we need to export `aam` then we should give it a proper name, e.g. `fromAcyclic`, and add docs/tests. See, for example:

https://github.com/snowleopard/alga/blob/07fa7cdfc5358a39fbf9bff47568d2fedf67f203/src/Algebra/Graph/Relation/Symmetric.hs#L98-L106

Reply:
Exporting it isn't necessary now. I'll remove it. Maybe I'll add it later if required.